sd/api_views.py

```diff
@@ -288,14 +296,24 @@ def get(self, request, format=None):
             author = Author.objects.get(uuid=userUUID)
 
             serverAuthors = Author.objects.filter(host=author.host)
+            print(serverAuthors)
 
-            serverPosts = Post.objects
+            serverPostUUIDs = []
             for author in serverAuthors:
-                serverPosts.union(Post.objects.filter(
-                    author=author.uuid).filter(visibility='5'))
+                temp = Post.objects.filter(author=author.uuid, visibility='5')
+                # print('temp ok')
+                for post in temp:
+                    print(post.uuid)
+                    print(Post.objects.get(uuid=post.uuid))
+                    filteredPosts = filteredPosts.union(
+                        Post.objects.filter(uuid=post.uuid))
```

Comment:
what is this doing? temp = posts by some author, then you're finding posts by their uuids, but you already have them?